scripts/apps/publish/directives/SubscribersDirective.ts

```diff
@@ -379,6 +379,37 @@ export function SubscribersDirective(
                     ));
                 }
             };
+
+            /**
+             * Function to generate and set appropriate schedule/label messages
+            */
+            $scope.getScheduleOrStatusLabel = function(subscriber) {
+                if (!subscriber) return '';
+
+                const hasSchedule = subscriber.schedule?.start_date && subscriber.schedule?.end_date;
+
+                if (hasSchedule) {
+                    const {start_date, end_date} = subscriber.schedule;
+                    const now = startOfDay(new Date());
+                    const mStart = startOfDay(new Date(start_date));
```

Comment:
Do not prefix variables with letters. I assume `mStart` was intended for moment dates, but it's not moment anymore. Even if it was moment - typescript type should be added instead of adding that information to the variable name.